config.go

```diff
@@ -397,6 +402,19 @@ func WriteBufferSize(size int) WriterOption {
 	return writerOption(func(config *WriterConfig) { config.WriteBufferSize = size })
 }
 
+// MaxRowsPerRowGroup configures the maximum number of rows that a writer will
+// produce in each row group.
+//
+// This limit is useful to control size of row groups in both number of rows and
+// byte size. While controlling the byte size of a row group is difficult to
+// achieve with parquet due to column encoding and compression, the number of
+// rows remains a useful proxy.
+//
+// Defaults to unlimited.
+func MaxRowsPerRowGroup(numRows int64) WriterOption {
+	return writerOption(func(config *WriterConfig) { config.MaxRowsPerRowGroup = numRows })
```

Comment:
what if numRows <= ?

Reply:
Do you need this case to be handled?

Reply:
No, just accidentally set zero and got infinite loop inside your lib

Reply:
How to get permissions to push branches ?

Reply:
We generally do not give push permissions to external contributors, however you can follow a typical github workflow:
- fork the repository on your own github account
- make the changes to your fork
- open a pull requests against segmentio/parquet-go from your fork

At which point we can review the changes and merge them in once we all agree on the contribution.

Reply:
Thanks! 
Offtop: is there a way to copy a parquet file just with another row group limit (max rows per rows group)

Reply:
Yes, you can configure the writer to generate the new parquet file with a different row group limit and copy rows from the previous file(s) which will rewrite all the rows and apply the new limits.